pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -1040,3 +1060,25 @@ func (m *ManagerImpl) setPodPendingAdmission(pod *v1.Pod) {
 
 	m.pendingAdmissionPod = pod
 }
+
+func (m *ManagerImpl) checkContainerAlreadyRunning(podUID, cntName string) bool {
+	if m.sourcesReady.AllReady() {
+		// initialization done, let's stop any guessing
+		return false
+	}
```

Comment:
This doesn't seem right. Can you please clarify? IIUC, sources are reflective of readiness of kubelet configuration sources.  IIUC we should return with `false` if the sources are not ready i.e. if  `!m.sourcesReady.AllReady()` evaluates to true.

Reply:
Probably the name is misleading, or I should move this check in the calling site. The idea here is that this logic should be triggered only during initialization, so we should skip it alltogether once initialization is complete. Using `AllReady` is (AFAIK) the established way to check if initialization is still ongoing or not.